### PR TITLE
Fix/ avatar label sizes

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Fix Avatar label sizes
WOAUILIB-3468

## Changelog
Fix Avatar label sizes

### CODE REVIEW

File: src/helpers/AvatarHelper.ts [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+      break;
+    case size < 100:
+      typography = Typography.text50H;
+      break;
+    default:
+    case size >= 100:
+      typography = Typography.text40H;
+      break;
+  }
+  return typography;
```
Comment (M-i-k-e-l) [inline src/helpers/AvatarHelper.ts:29]: I like this feature! Please add it to the dev sync.
I assume you've talked about the UX on this change, right?

File: demo/src/screens/__tests__/__snapshots__/AvatarScreen.spec.js.snap
Comment (M-i-k-e-l) [inline demo/src/screens/__tests__/__snapshots__/AvatarScreen.spec.js.snap:1]: You'll probably need to fix private as well

File: src/helpers/AvatarHelper.ts (hunk 1 above)
Comment (lidord-wix) [reply src/helpers/AvatarHelper.ts:29]: Yes with Diana :)

File: demo/src/screens/__tests__/__snapshots__/AvatarScreen.spec.js.snap
Comment (lidord-wix) [reply demo/src/screens/__tests__/__snapshots__/AvatarScreen.spec.js.snap:1]: Yup

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**